data_validation/cli_tools.py

```diff
@@ -1062,8 +1062,16 @@ def get_validation(validation_name, config_dir=None):
         mgr = state_manager.StateManager(file_system_root_path=config_dir)
         return mgr.get_validation_config(validation_name, config_dir)
     else:
-        mgr = state_manager.StateManager()
-        return mgr.get_validation_config(validation_name)
+        if validation_name.startswith("gs://"):
```

Comment:
I think this logic should go within the StateManager get_validation_config() function. In the function, we can check for `if validation_name.startswith("gs://")` similar to [this function](https://github.com/GoogleCloudPlatform/professional-services-data-validator/blob/47154b4139bf22358359c53fe25bbce44745589f/data_validation/state_manager.py#L147).